<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="backgroundWid" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#backgroundWid{
background-color:qlineargradient(spread:pad, x1:0.477833, y1:0, x2:0.507, y2:0.988636, stop:0 rgba(195, 236, 192, 255), stop:0.522167 rgba(168, 198, 232, 255))}
</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>351</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 48pt &quot;Avenir Next&quot;;color: rgb(255, 255, 255);
color: rgb(10, 6, 17);</string>
     </property>
     <property name="text">
      <string>Manage Users</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>711</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(10, 12, 23);
font: 23pt &quot;Hoefler Text&quot;;</string>
     </property>
     <property name="text">
      <string>Here you can add, list, update, or delete users who have created an account</string>
     </property>
    </widget>
    <widget class="QPushButton" name="update_btn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>290</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(8, 9, 14);
font: 24pt &quot;Georgia&quot;;</string>
     </property>
     <property name="text">
      <string>Update this User</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_btn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>340</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(8, 9, 14);
font: 24pt &quot;Georgia&quot;;</string>
     </property>
     <property name="text">
      <string>Delete this User</string>
     </property>
    </widget>
    <widget class="QLabel" name="user_le">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(16, 10, 32);
font: 16pt &quot;Charter&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailLineEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>250</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0)</string>
     </property>
    </widget>
    <widget class="QLabel" name="pass_le">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>250</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(16, 10, 32);
font: 16pt &quot;Charter&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="cancel_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(9, 6, 18)</string>
     </property>
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="user_le_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(16, 10, 32);
font: 16pt &quot;Charter&quot;;</string>
     </property>
     <property name="text">
      <string>User:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
